library/agent/hooks/wrapRequire.ts

```diff
@@ -188,11 +188,6 @@ function patchPackage(this: mod, id: string, originalExports: unknown) {
     .map((pkg) => pkg.getVersions())
     .flat();
 
-  // We don't want to patch this package because we do not have any hooks for it
-  if (!versionedPackages.length) {
-    return originalExports;
-  }
-
   // Read the package.json of the required package
   const packageJson = originalRequire(
```

Comment:
Maybe we should wrap this in a try ... catch block then, as we run this for all modules now, so that Zen does not crash in case the package.json file does not exist for some unknown reason? (not sure if this is even possible)

Reply:
Added a try/catch to be safe!